api/proto/teleport/legacy/types/types.proto

```diff
@@ -4608,6 +4608,33 @@ message DatabaseServiceV1 {
 message DatabaseServiceSpecV1 {
   // ResourceMatchers is the configured match for Database resources.
   repeated DatabaseResourceMatcher ResourceMatchers = 1 [(gogoproto.jsontag) = "resources"];
+  // InstanceMetadata contains metadata from the host that is running the service.
+  InstanceMetadata InstanceMetadata = 2 [(gogoproto.jsontag) = "instance_metadata"];
```

Comment:
I would avoid the ..Metadata naming. The struct can because a bin for all the fields. What would you say about renaming it to something like `CloudIdentity` with the [AWS,Azure,GCP] membes ?